Orange/widgets/data/owtransform.py

```diff
@@ -97,8 +119,8 @@ def apply(self):
     def send_report(self):
         if self.data:
             self.report_data("Data", self.data)
-        if self.template_domain is not None:
-            self.report_domain("Template data", self.template_domain)
+        if self.template_data is not None:
```

Comment:
If self.template_data is not None, it evaluates to True (the is not None part is not necessary).